src/actions/adjust/ImproveAction.ts

```diff
@@ -2,18 +2,32 @@ import Action from "../Action";
 import Qualifier from "../../qualifier/Qualifier";
 import QualifierValue from "../../qualifier/QualifierValue";
 
+/**
+ * @description Defines how to improve an image by automatically adjusting image colors, contrast and brightness.</br>
```

Comment:
This is just a detail but I have been keeping a new line in between the description and learn more

Reply:
This has a new line, (using </br>) or did you mean something else?